src/stitching/makeRemoteExecutableSchema.ts

```diff
@@ -49,6 +48,27 @@ export type FetcherOperation = {
   context?: { [key: string]: any };
 };
 
+/**
+ * This type has been copied inline from its source on `@types/graphql`:
+ *
+ * https://git.io/Jv8NX
```

Comment:
`@types/graphql` is no longer being kept up, but it seems the same type is available in the source still? We could also copy/reference from there if not, so the reference is current.

https://git.io/Jv4tt

Reply:
Aside from the name, the reference is current.  Referencing it from the new location could be a problem for cross-graphql version compatibility, so keeping it in line here seems okay for now.